ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/df_driver.py

```diff
@@ -191,35 +225,57 @@ def parse_and_process_data_preprocessing_config(object_storage_client, spark, ge
             data_asset_detail.pop('prefix')
             for object_details in objects_details.objects:
                 if object_details.name.endswith('.csv'):
+                    prefix = object_details.name.split('/')[0]
+                    if prefix not in preprocessed_data_prefix_to_column:
+                        continue
+                    columns = preprocessed_data_prefix_to_column[prefix]
                     data_asset_detail['object'] = object_details.name
+                    output_processed_data_info.append({
```

Comment:
The earlier way of extracting object-storage object's details from output path is because the information is only available as object-storage object's url, but now looks like the information is available in `contents["stagingDestination"]` object. Please directly use those values and avoid redundant computations.